docker/Dockerfile

```
@@ -1,19 +1,17 @@
-FROM continuumio/miniconda3
+FROM esmvalgroup/esmvalcore:latest
```

```suggestion
FROM continuumio/miniconda3
```
I would just keep the miniconda base image, because updating conda environments is asking for trouble

---

The question is, do we want to keep master compatible with the latest release of ESMValCore or with the current ESMValCore master branch? I assumed the later, so I started from it

---

All recipes and diagnostics in the master branch work with the latest released version of ESMValCore, because we want to provide diagnostic developers with a stable development environment.